PokedexV2/Views/PokemonFeatures.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  PokemonFeatures.swift
+//  PokedexV2
+//
+//  Created by JonathanA on 19/10/22.
+//
+
+import SwiftUI
+import Kingfisher
+
+struct PokemonFeatures: View {
+    @ObservedObject var vm = PokemonViewModel()
+    var pokemonId: Int
+    @Binding var isActive: Bool
+    var body: some View {
+        ZStack {
+            Color.white
+                .ignoresSafeArea()
+            VStack {
+                HStack{
+                    Button(action: {
+                        isActive = false
+                    }) {
+                        Image(systemName: "arrowshape.turn.up.backward.fill")
+                            .foregroundColor(Color("primary"))
+                    }
+                    .padding()
+                    Spacer()
+                }
+                ScrollView {
+                    if let pokemonDetail = vm.pokemonDetails{
```

Comment:
Instead of pokemonDetails is pokemonDetails

Reply:
it's done